openshift/rh-che.app.yaml

```diff
@@ -120,6 +120,16 @@ objects:
             value: "openshift"
           - name: SENTRY_RELEASE
             value: ${IMAGE_TAG}
+          - name: CHE_CORE_JSONRPC_PROCESSOR__MAX__POOL__SIZE
```

Comment:
We do not add env vars that are in config map here anymore. We include all the entries from the config map as env vars, so, please, just rename config maps entries to be appropriate env vars for Che

Reply:
:+1:

Reply:
fixed.